docs/reference/cli/filters.md

```diff
@@ -207,6 +207,61 @@ at least one of the following label sets:
 Note: the LLM prompt includes a subset of your Prometheus relabeling rules to
 avoid using too many tokens, so you may not see every label set in the output.
 
+## AWS
+
+Activate the AWS integration:
+```bash
+k8sgpt integration activate aws
+```
+Once activated, you should see the following success message displayed:
+```
+Activated integration aws
+```
+
+This will activate the AWS Kubernetes Operator into the Kubernetes cluster and make it possible for K8sGPT to interact with the results of the Operator.
+
+Once the AWS integration is activated inside the cluster, K8sGPT will have access to EKS:
+```bash
+❯ k8sgpt filters list
+
+Active:
+> StatefulSet
+> Ingress
+> Pod
+> Node
+> ValidatingWebhookConfiguration
+> Service
+> EKS (integration)
+> PersistentVolumeClaim
+> MutatingWebhookConfiguration
+> CronJob
+> Deployment
+> ReplicaSet
+Unused:
+> Log
+> GatewayClass
+> Gateway
+> HTTPRoute
+> HorizontalPodAutoScaler
+> PodDisruptionBudget
+> NetworkPolicy
+```
+
+More information can be found on the official [AWS-Operator documentation](https://aws.amazon.com/blogs/opensource/aws-service-operator-kubernetes-available/).
```

Comment:
Could you please add more information on what this operator is and how it relates to the previous section? For someone who has never looked at the integration between AWS and K8sGPT this might be a little confusing

Reply:
Hey @AnaisUrlichs, thank you for reviewing this PR. I have added descriptions as per your suggestion.